Source/JavaScriptCore/dfg/DFGSafeToExecute.h

```diff
@@ -414,6 +414,7 @@ bool safeToExecute(AbstractStateType& state, Graph& graph, Node* node, bool igno
         return node->arrayMode().alreadyChecked(graph, node, state.forNode(graph.child(node, 0)));
 
     case MultiGetByVal:
+    case MultiPutByVal:
```

Comment:
Seems like these could move if it were safe to do all the equivalent Put/GetByVals? It seems like it's worth a comment at least saying that for posterity.

Reply:
Added comment.